model.js

```diff
@@ -0,0 +1,9 @@
+const db = require(`${__dirname}/db/connection`);
+
+function fetchTopics () {
+    return db.query(`SELECT * FROM topics;`)
+        .then(({ rows: topics }) => {return topics})
+        .catch((err) => {Promise.reject(err)})
```

Comment:
the model doesn't necessarily need a catch block as the controller will handle any errors